.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Build Quarkus master
         run: git clone https://github.com/quarkusio/quarkus.git && cd quarkus && mvn -B clean install -DskipTests -DskipITs -DskipDocs
       - name: Build with Maven
-        run: mvn -fae -V -B clean test -Dquarkus-core-only
+        run: mvn -fae -V -B clean test -Dquarkus-core-only -Dinclude.serverless
```

Comment:
Out of curiosity, why cannot this module be executed as part of the whole suite?

Reply:
because at the build phase the extension tries to communicate with OCP server and expects knative / OpenShift Serverless to be installed

And that's also why we end at `test` phase and not `package` phase in GH CI jobs